pkg/deploy/gateway/gateway.go

```diff
@@ -252,11 +252,18 @@ func GetGatewayRouteConfig(deployContext *deploy.DeployContext, component string
 	pathRewrite := pathPrefix != "/" && stripPrefix
 	nativeUser := util.IsNativeUserModeEnabled(deployContext.CheCluster)
 
+	var rule string
+	if component == "che-gateway-route-server" {
```

Comment:
what about just passing an array of `pathPrefix`es instead of this?

Reply:
yes, issue is that `/` has to be Path, not PathPrefix, so probably it will have to be some custom struct or construct it earlier... Also logical operators and parenthesis complicate it even more https://doc.traefik.io/traefik/routing/routers/#rule. I'm not sure how future-proof do we want to be here.